js/functions.js

```diff
@@ -2551,7 +2551,14 @@ function getEventDescription(isAerobatics, locationName, minutes) {
                             $("#aircraftImg").attr("src",`icons/genericAircraft.svg`);
                         }
                         $("#gottoVoiceMessagePopup")[0].style.display = "block";
-                        $("#aircraftName").html(`${aircraft.type} - ${aircraft.name}`);
+                        //Change type and name if efroni to aerobatic team 
+                        if (aircraft.aircraftTypeId == '25'){
```

Comment:
```suggestion
                        if (aircraft.name == 'עפרוני'){
```
א. נראה לי יותר קריא
ב. אני לא אוהב שיש התייחסות לID ספציפיים בקוד